DevVehicle35-Motors/MiniBus.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
```

Comment:
We agree that our Classes should be placed on a directory called models

Reply:
ok, I moved the File